spec/lib/data_update_scripts/add_fastly_http_purge_feature_flag_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "rails_helper"
+require Rails.root.join(
+  "lib/data_update_scripts/20210209152726_add_fastly_http_purge_feature_flag.rb",
+)
+
+describe DataUpdateScripts::AddFastlyHttpPurgeFeatureFlag do
+  after do
+    FeatureFlag.remove(:fastly_http_purge)
+  end
+
+  it "adds the :fastly_http_purge flag", :aggregate_failures do
+    expect(FeatureFlag.exist?(:fastly_http_purge)).to be(false)
```

Comment:
We could change (no pun intended) this to use a `change` assertion:

```ruby
expect do
  described_class.new.run
end.to change { FeatureFlag.exist?(:fastly_http_purge) }.from(true).to(false)
```

Reply:
thanks!